test/new-e2e/go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	// Commit sha matches the first 8 digits of the commit sha in the module version
 	// Example: 	github.com/DataDog/test-infra-definitions v0.0.0-YYYYMMDDHHmmSS-0123456789AB
 	// => image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:01234567
-	github.com/DataDog/test-infra-definitions v0.0.0-20230619131020-fb813362e3e0
+	github.com/DataDog/test-infra-definitions v0.0.0-20230622123358-60eb38e90955
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Will update the comment, did not know we can use the full commit sha as runner tag

Reply:
I added dual tagging to be easier.

Reply:
Nice !